platformio.ini

```
@@ -40,6 +40,7 @@ lib_deps =
     wjtje/qr-code-generator-library@1.7.0  ; qr_code
     functionpointer/arduino-MLX90393@1.0.0 ; mlx90393
     pavlodn/HaierProtocol@0.9.31           ; haier
+    kikuchan98/pngle@1.0.2                 ; PNG decoder
```

This should be the component using it
```suggestion
    kikuchan98/pngle@1.0.2                 ; online_image
```